<launch>
	<node name="stage_sim" pkg="stage_ros" type="stageros" args="$(find read_sensor_data)/world/circularTrajectory.world" output="screen"/>
  
	<node name="robot0" pkg="read_sensor_data" type="readOdomScan" output="screen">
		<remap from="/robot0/goal" to="/goal"/>
        <remap from="/pose" to="/robot_0/base_pose_ground_truth"/>
		<remap from="/scan" to="/robot_0/base_scan"/>
		<remap from="/cmd_vel" to="/robot_0/cmd_vel"/>
        <remap from="/pedirSiguienteGoal" to="/robot_0/pedirSiguienteGoal"/>
	</node>

    <node name="robot1" pkg="read_sensor_data" type="readOdomScan" output="screen">
        <remap from="/robot1/goal" to="/goal"/>
		<remap from="/pose" to="/robot_1/base_pose_ground_truth"/>
		<remap from="/scan" to="/robot_1/base_scan"/>
		<remap from="/cmd_vel" to="/robot_1/cmd_vel"/>
        <remap from="/pedirSiguienteGoal" to="/robot1/pedirSiguienteGoal"/>
	</node>


    <node name="robot2" pkg="read_sensor_data" type="readOdomScan" output="screen">
        <remap from="/robot2/goal" to="/goal"/>
		<remap from="/pose" to="/robot_2/base_pose_ground_truth"/>
		<remap from="/scan" to="/robot_2/base_scan"/>
		<remap from="/cmd_vel" to="/robot_2/cmd_vel"/>
        <remap from="/pedirSiguienteGoal" to="/robot_2/pedirSiguienteGoal"/>
	</node>

	
</launch>